packages/api/package.json

```
@@ -23,7 +23,7 @@
     "@sentry/node": "8.32.0",
     "@sentry/profiling-node": "8.32.0",
     "axios": "1.5.1",
-    "bakosafe": "0.0.90-beta",
+    "bakosafe": "0.0.92-beta",
```

```suggestion
    "bakosafe": "0.0.92-beta",
```

This NPM version does not exist, therefore have not done an install. We need to:
- Review PR
- Release the following PR 
- Then perform `pnpm install` on this PR